modules/cpmso-creating-cr.adoc

```diff
@@ -6,4 +6,88 @@
 [id="cpmso-creating-cr_{context}"]
 = Creating a control plane machine set custom resource
 
-To use the Control Plane Machine Set Operator on clusters that do not generate a `ControlPlaneMachineSet` custom resource (CR), you must create the CR manually.
\ No newline at end of file
+To use the Control Plane Machine Set Operator, you must ensure that a `ControlPlaneMachineSet` custom resource (CR) with the correct settings for your cluster exists. On a cluster without a generated CR, you must create the CR manually and activate the Operator.
+
+[NOTE]
+====
+For more information about the structure and parameters of the CR, see "Control Plane Machine Set Operator configuration".
+====
+
+.Procedure
+
+. Create a YAML file using the following template:
++
+--
+.Control plane machine set CR YAML file template
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster
+  namespace: openshift-machine-api
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <cluster_id> <1>
+      machine.openshift.io/cluster-api-machine-role: master
+      machine.openshift.io/cluster-api-machine-type: master
+  state: Active <2>
+  strategy:
+    type: RollingUpdate <3>
+  template:
+    machineType: machines_v1beta1_machine_openshift_io
+    machines_v1beta1_machine_openshift_io:
+      failureDomains:
+        platform: <platform> <4>
+        <platform_failure_domains> <5>
+      metadata:
+        labels:
+          machine.openshift.io/cluster-api-cluster: <cluster_id> <6>
+          machine.openshift.io/cluster-api-machine-role: master
+          machine.openshift.io/cluster-api-machine-type: master
+      spec:
+        providerSpec:
+          value:
+            <platform_provider_spec> <7>
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. You must specify this value when you create a `ControlPlaneMachineSet` CR. If you have the OpenShift CLI (`oc`) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+<2> Specify the state of the Operator. When the state is `Inactive`, the Operator is not operational. You can activate the Operator by setting the value to `Active`. 
++
+[IMPORTANT]
+====
+Before you activate the Operator, you must ensure that the CR configuration is correct for your cluster requirements.
+====
+<3> Specify the update strategy for the cluster. The allowed values are `OnDelete` and `RollingUpdate`. The default value is `RollingUpdate`. 
+//For more information about update strategies, see "Updating the control plane configuration".
```

Comment:
Also wasn't sure if this comment tag was meant to be removed

Reply:
This is still placeholder content for me until I have related references done. I'm gonna stick a `todo` in front of it though to make it obvious (and harder for me to forget later).